drv/gimlet-seq-server/src/main.rs

```diff
@@ -1231,7 +1231,7 @@ fn read_spd_data_and_load_packrat(
                 spd.read_into(&mut tmp[chunk..])
             })?;
 
-            packrat.set_spd_eeprom(ndx, true, 0, &tmp);
+            packrat.set_spd_eeprom(ndx, spd::PAGE_SIZE, &tmp);
```

Comment:
This and the other change in the file look weird, or at least asymmetric. I think this is because the `bool` before switched which page it was and instead this is just an offset?